src/RedisCache/RedisCache/Commands/SetAzureRedisCache.cs

```diff
@@ -68,6 +68,12 @@ public class SetAzureRedisCache : RedisCacheCmdletBase
         [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "A hash table which represents tags.")]
         public Hashtable Tag { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Specifies the type of identity used for the Azure Cache for Redis. Valid values: \"SystemAssigned\" or \"UserAssigned\" or \"SystemAssignedUserAssigned\" or \"None\" ")]
+        public string IdentityType { get; set; }
```

Comment:
Please provide a argument completer for IdentityType

Reply:
added